_posts/2022-11-03-autoscaling-functions.md

```diff
@@ -0,0 +1,212 @@
+---
+title: "Autoscaling for OpenFaaS Functions"
+description: "Learn how and why to use autoscaling for OpenFaaS Functions"
```

Comment:
We redesigned the autoscaler in OpenFaaS to match the patterns we were seeing for customer workloads.